testing/shairport-sync/shairport-sync.initd

```diff
@@ -0,0 +1,13 @@
+#!/sbin/openrc-run
+
+description="Implements a synchronous (multi-room-capable) AirPlay receiver"
+name="shairport-sync $SVCNAME"
```

Comment:
`$SVCNAME` is deprecated, it should be named `$RC_SVCNAME`. Please read [openrc-run(8)](http://manpages.org/openrc-run/8).